recipes/pango/all/conanfile.py

```diff
@@ -31,50 +31,48 @@ class PangoConan(ConanFile):
         "with_xft": [True, False],
         "with_freetype": [True, False],
         "with_fontconfig": [True, False],
+        "with_introspection": [True, False],
     }
     default_options = {
         "shared": False,
         "fPIC": True,
         "with_libthai": False,
         "with_cairo": True,
-        "with_xft": False,
-        "with_freetype": False,
-        "with_fontconfig": False,
+        "with_xft": True,
+        "with_freetype": True,
+        "with_fontconfig": True,
+        "with_introspection": False,
     }
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
-
-        if self.settings.os in ["FreeBSD", "Linux"]:
-            self.options.with_xft = True
-        if not self.settings.os in ["Macos", "Windows"]:
-            self.options.with_freetype = True
-            self.options.with_fontconfig = True
+        if self.settings.os not in ["FreeBSD", "Linux"]:
+            del self.options.with_xft
+        if self.settings.os in ["Macos", "Windows"]:
+            self.options.with_freetype = False
+            self.options.with_fontconfig = False
 
     def configure(self):
         if self.options.shared:
             self.options.rm_safe("fPIC")
         self.settings.rm_safe("compiler.libcxx")
         self.settings.rm_safe("compiler.cppstd")
+        self.options["cairo"].with_freetype = self.options.with_freetype
+        self.options["cairo"].with_fontconfig = self.options.with_fontconfig
```

Comment:
Could you explain why are you adding this? In the previous revision it was working with cairo using both `freetype` and `fontconfig` enabled. 

Do you have any reference from upstream? 

I am trying to fix the missing binaries issues :(

Reply:
Well, how do you feel about reverting the change to maintain the previous version from Conan Center Index? The idea is to avoid the missing binaries issues associated with this change.

Reply:
> The idea is to avoid the missing binaries issues associated with this change.

Both the `cairo` and `pango` would have the `with_freetype` and `with_fontconfig` options set to `True` as a result of this PR, so I don't see how it would cause issues in the default case at least.

I'll just revert it since I don't care about the change too much.